app/views/welcome/_next-event.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class='main'>
+  <h1> Next Event </h1>
+  <% if !next_event %>
```

Comment:
Agreed! You can always flip a negative logical statement to be positive. It is usually easier for humans to understand: 
`if next_event.nill?` than it is to understand ` if !next_event`.

In this you also get to choose which code goes in the `if` and which code goes in the `else`